src/main/java/com/bootcamp/demo/Controller/CardController.java

```diff
@@ -0,0 +1,21 @@
+package com.bootcamp.demo.Controller;
+
+import com.google.api.core.ApiFuture;
+import com.google.cloud.firestore.Firestore;
+import com.google.cloud.firestore.WriteResult;
+import com.google.firebase.cloud.FirestoreClient;
+import org.springframework.web.bind.annotation.*;
+
+
+@RestController
+@RequestMapping("/card")
+public class CardController {
+    private Firestore firestoreDB;
```

Comment:
Please make this field `final` so you'll get an immutable Bean, and create a public constructor so all dependencies are injected through that constructor.